api/Range.json

```diff
@@ -449,7 +449,8 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": false
+              "version_added": "1",
```

Comment:
Interesting! This entry should be remove, but not in this PR, and having this data is a good clue for when it is removed.